scrapers/errata.py

```diff
@@ -113,19 +113,28 @@ def update_neo4j(self, advisories):
                     nvr = '-'.join((associated_build['name'], associated_build['version'],
                                     associated_build['release']))
 
-                    sql = r"""\
+                    sql = """\
                         SELECT created_at
                         FROM Errata_public.errata_brew_mappings
                         INNER JOIN Errata_public.brew_builds
                         ON Errata_public.errata_brew_mappings.brew_build_id
                             = Errata_public.brew_builds.id
-                        WHERE errata_id = {0} AND current = 1 AND nvr = {1};
+                        WHERE errata_id = {0} AND current = 1 AND nvr = '{1}';
                     """.format(advisory['id'], nvr)
                     log.info('Getting the time build {0} was added to advisory {1}'.format(
                         nvr, advisory['id']))
-                    time_attached = self.teiid.query(sql)
-
-                    adv.attached_builds.connect(build, {'time_attached': time_attached})
+                    time_attached_result = self.teiid.query(sql)
+                    if time_attached_result:
+                        time_attached = time_attached_result[0].get('created_at')
```

Comment:
Wouldn't the SQL query already grab just the 'created_at'?

Reply:
It returns a list with a single entry. You can see that in the documentation:
https://estuary-api.readthedocs.io/en/latest/module_documentation/scrapers.html#scrapers.teiid.Teiid.query